docs/jwst/rscd/reference_files.rst

```diff
@@ -43,7 +57,7 @@ It uses ``EXTNAME=RSCD`` and contains seventeen columns:
 * ILLUM_ZP: float
 * ILLUM_SLOPE: float
 * ILLUM2: float
-* PARAM3: :math:`b{3}' in equation
+* PARAM3: :math:`b{3}` in equation
 * CROSSOPT: float, crossover point
 * SAT_ZP: float
 * SAT_SLOPE: float
```

Comment:
Are there short (1-3 word) descriptions available for columns like ILLUM_ZP, ILLUM_SLOPE, ILLUM2, SAT_ZP, SAT_SLOPE?